# HG changeset patch
# User Doug Stewart  <doug.dastew@gmail.com>
# Date 1343393679 14400
#      Fri Jul 27 08:54:39 2012 -0400
# Node ID bd84bddeee7a5eca9b58056a6a7f336d4c14f740
# Parent  86a95d6ada0d93e2dd4ac99add2c4e836cf8e4df
Added one more *seealso* in "I" and removed a space at start of line, all in data.cc

diff --git a/src/data.cc b/src/data.cc
--- a/src/data.cc
+++ b/src/data.cc
@@ -4309,26 +4309,27 @@ Return a scalar, matrix, or N-dimensiona
 to the pure imaginary unit, defined as\n\
 @tex\n\
 $\\sqrt{-1}$.\n\
 @end tex\n\
 @ifnottex\n\
 @code{sqrt (-1)}.\n\
 @end ifnottex\n\
 \n\
- I, and its equivalents i, J, and j, are functions so any of the names may\n\
+I, and its equivalents i, J, and j, are functions so any of the names may\n\
 be reused for other purposes (such as i for a counter variable).\n\
 \n\
 When called with no arguments, return a scalar with the value @math{i}.  When\n\
 called with a single argument, return a square matrix with the dimension\n\
 specified.  When called with more than one scalar argument the first two\n\
 arguments are taken as the number of rows and columns and any further\n\
 arguments specify additional matrix dimensions.\n\
 The optional argument @var{class} specifies the return type and may be\n\
 either \"double\" or \"single\".\n\
+@seealso{log, exp, e, pi, i, j}\n\
 @end deftypefn")
 {
   return fill_matrix (args, Complex (0.0, 1.0), "I");
 }
 
 DEFALIAS (i, I);
 DEFALIAS (J, I);
 DEFALIAS (j, I);
